PowerEditor/src/NppIO.cpp

```diff
@@ -2179,15 +2178,14 @@ bool Notepad_plus::fileRenameUntitledPluginAPI(BufferID id, const wchar_t* tabNe
 	bool isSnapshotMode = NppParameters::getInstance().getNppGUI().isSnapshotMode();
 	if (isSnapshotMode)
 	{
-		wstring oldName = buf->getFullPathName();

```

Comment:
The local variable should be declared in the scope where it is used.
Is there any reason to move out `oldName` from its scope?

Reply:
>Is there any reason to move out oldName from its scope?

Yes.

`wstring oldName = buf->getFullPathName();`

needs to be BEFORE:

`buf->setFileName(tabNewNameStr.c_str());`

otherwise, the "old name" is lost.

Reply:
@alankilborn 
Oups! You're right about it!